libs/barista-components/filter-field/src/filter-field.html

```diff
@@ -95,6 +99,19 @@
       [unit]="_currentDef && !!_currentDef!.range && _currentDef!.range!.unit"
       (rangeSubmitted)="_handleRangeSubmitted($event)"
     ></dt-filter-field-range>
+
+    <dt-filter-field-multi-select
+      #multiSelect="dtFilterFieldMultiSelect"
+      (multiSelectSubmitted)="_handleMultiSelectSubmitted($event)"
+      [optionsOrGroups]="
+        _currentDef &&
+        !!_currentDef!.multiSelect &&
+        _currentDef!.multiSelect!.multiOptions
+      "
+      [inputValue]="_inputValue"
+      panelWidth="400px"
```

Comment:
this is gonna be a problem soon, but i am fine with this being in there for v1

Reply:
Autocomplete panel is also somewhere clipped. Let's see when it will be a problem.